rest_framework-stubs/schemas/coreapi.pyi

```diff
@@ -53,11 +53,11 @@ class AutoSchema(ViewInspector):
     def get_filter_fields(self, path: str, method: str) -> List[Dict[str, Any]]: ...
     def get_manual_fields(self, path: str, method: str) -> List[coreapi.Field]: ...
     @staticmethod
-    def update_fields(fields: List[coreapi.Fields], update_with: List[coreapi.Fields]) -> List[coreapi.Fields]: ...
+    def update_fields(fields: List[coreapi.Field], update_with: List[coreapi.Field]) -> List[coreapi.Field]: ...
```

Comment:
Maybe this line is the cause of the problem? https://github.com/typeddjango/djangorestframework-stubs/blob/master/tests/plugins.ini#L2